docs/blog/2019-05-29-intergrating-graphiql-explorer/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Improvements to GraphiQL IDE - adding GraphiQL Explorer
+date: 2019-05-29
+author: Michal Piechowiak
+tags:
+  - graphql
+  - graphiql-explorer
+  - developer experience
+---
+
+Gatsby's data layer is powered by [GraphQL](https://graphql.org/). This means that if you are building a Gatsby site, you will almost certainly use GraphQL to take advantage of Gatsby's rich plugin ecosystem that extends this schema with data from _anywhere_. A popular tool for exploring your GraphQL schema is [GraphiQL](https://github.com/graphql/graphiql) — which is a graphical, interactive, in-browser GraphQL development environment. It serves as an interactive playground where you can compose, test, and see the live results of your queries. If you haven't seen or used GraphiQL, it looks something like this:
+
+<video controls="controls" autoplay="true" loop="true">
```

Comment:
I'm also wondering if we could wrap these in a bit more markup and add some text alternatives:

```suggestion
<figure>
    <video controls="controls" autoplay="false" loop="true">
        <source />
        <p />
    </video>
    <figcaption>some kind of caption here</figcaption>
</figure>
```

Reply:
wrapped videos in figure + figcaption